src/main/java/com/salam/task_manager/dto/TaskDto.java

```diff
@@ -0,0 +1,22 @@
+package com.salam.task_manager.dto;
+
+import lombok.*;
+
+import java.time.ZonedDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
```

Comment:
Why so many annotation ?
describe what each does

Reply:
It is preferable to have a requestDto for requests and a responseDto and responses

Reply:
no answer?

Reply:
switched to requestDto and responseDto in a new commit [https://github.com/darthmal/Task-Manager-Spring-boot-project/pull/1/commits/94b7376d5caddc1942873ecb8577f5e20a2d3ce6](url)